cloud/aws-functions/orphaned_openshift_instances.py

```diff
@@ -0,0 +1,59 @@
+# Remove openshift resources.
+import logging
+import datetime
+import boto3
+import sys
+from time import sleep
+from botocore.exceptions import ClientError
+from boto3.exceptions import Boto3Error
+
+
+def isInstanceToTerminate(instance):
+    tags = instance.tags
+    tags_dict = {item['Key']: item['Value'] for item in tags}
+    state = instance.state['Name']
+    if 'team' not in tags_dict.keys() or state != 'running':
+        return False
+    if 'delete-cluster-after-hours' not in tags_dict.keys() and tags_dict['team'] == 'cloud':
```

Comment:
maybe we can return False if tags_dict['team'] != 'cloud' don't care about team after